.github/workflows/licensed.yml

```diff
@@ -0,0 +1,54 @@
+name: licenses check
+
+on:
+  push:
+    branches:
+      - main
+    tags-ignore: "**"
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+
+# github/licensed-ci
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  licensed:
+    name: licensed check
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Install Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: go.mod
+
+      - run: go version
+
+      - run: go mod vendor
+
+      # Ruby is required for licensed
+      - uses: ruby/setup-ruby@bc1dd263b68cb5626dbb55d5c89777d79372c484
+        with:
+          ruby-version: "3.2"
+
+      - uses: github/setup-licensed@v1
+        with:
+          version: 4.x
+
+      - uses: github/licensed-ci@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          workflow: branch
+        env:
+          GOPRIVATE: "*github.com/github/*"
+          GOPROXY: "direct"
```

Comment:
Are these `env` values necessary? I don't see any private modules in `go.mod`

Reply:
done in https://github.com/actions/actions-sync/pull/94/commits/fc6c0a105afef7c6474259884fe2ad40d5453f0d